models/public/octave-densenet-121-0.125/octave-densenet-121-0.125.md

```diff
@@ -2,11 +2,8 @@
 
 ## Use Case and High-Level Description
 
-The `octave-densenet-121-0.125` model is a modification of `densenet-121` from [this paper](https://arxiv.org/pdf/1608.06993) with octave convolutions from [Drop an Octave: Reducing Spatial Redundancy in Convolutional Neural Networks with Octave Convolution](https://arxiv.org/abs/1904.05049) with `alpha=0.125`. As origin, it's designed to perform image classification. For details about family of octave convolution models, check out the [repository](https://github.com/facebookresearch/OctConv).
+The `octave-densenet-121-0.125` model is a modification of [`densenet-121`](https://arxiv.org/pdf/1608.06993) with Octave convolutions from [Drop an Octave: Reducing Spatial Redundancy in Convolutional Neural Networks with Octave Convolution](https://arxiv.org/abs/1904.05049) with `alpha=0.125`. The model is originally designed for image classification. For details about family of Octave Convolution models, check out the [repository](https://github.com/facebookresearch/OctConv).
```

Comment:
> The model is originally designed for image classification.

I think the intent of the original sentence was more along the lines of "Like the original model, this model is designed for image classification". @eizamaliev correct me if I'm wrong.

Reply:
Correct. In original sentence was made mistake, must be "As original", because this model is modification of another model. This is applicable for all `octave-*` models.